api-reference/beta/resources/directory.md

```diff
@@ -0,0 +1,57 @@
+# directory resource type
+
+Represents deleted items. Deleted items are commonly referred to as recycle bin items.
+
+Actions for the directory resource include the ability restore or permanently delete an item. Currently, the restore and permanent delete functionality is available for users and groups. The following scenarios are enabled for deleted items:
+
+1. An administrator can restore or permanently delete previously deleted user.
+2. If a user is an administrator of a group, they can restore or permanently delete the group.
```

Comment:
I believe your comment is correct.  Only admins can permanently delete any deleted items.